src/views/Auction/index.tsx

```diff
@@ -101,10 +103,15 @@ export function AuctionView() {
     return auctions
   })
 
-  const bids = useSelector<RootState, AuctionBid[]>(state => {
-    return state.BidReducer.bids
+  const bidsbySale = useSelector<RootState, any>(state => {
+    return state.BidReducer.bidsBySaleId[params.auctionId]
   })
```

Comment:
Chore: [use `camelCase` for functions and variable names](https://github.com/cryptonative-ch/mesa-interface/blob/main/CONTRIBUTING.md#naming-convention)